### ISSUE TITLE
CaptureScreen error

### ISSUE THREAD

**Issue by Divaite** [punker76/gong-wpf-dragdrop#219]
Hi.

First of all, thanks for the great library, works perfectly.
Unfortunately sometimes this error occurs:

```
"The parameter value must be greater than zero.\r\nParameter name: pixelWidth"
   at System.Windows.Media.Imaging.RenderTargetBitmap..ctor(Int32 pixelWidth, Int32 pixelHeight, Double dpiX, Double dpiY, PixelFormat pixelFormat)
   at GongSolutions.Wpf.DragDrop.DragDrop.CaptureScreen(Visual target, FlowDirection flowDirection, Double dpiX, Double dpiY) in d:\projects\git\gong-wpf-dragdrop\GongSolutions.Wpf.DragDrop\DragDrop.cs:line 502
   at GongSolutions.Wpf.DragDrop.DragDrop.CreateDragAdorner() in d:\projects\git\gong-wpf-dragdrop\GongSolutions.Wpf.DragDrop\DragDrop.cs:line 432
   at GongSolutions.Wpf.DragDrop.DragDrop.DropTarget_PreviewDragOver(Object sender, DragEventArgs e) in d:\projects\git\gong-wpf-dragdrop\GongSolutions.Wpf.DragDrop\DragDrop.cs:line 857
   at GongSolutions.Wpf.DragDrop.DragDrop.DropTarget_PreviewDragEnter(Object sender, DragEventArgs e) in d:\projects\git\gong-wpf-dragdrop\GongSolutions.Wpf.DragDrop\DragDrop.cs:line 825
   at System.Windows.RoutedEventArgs.InvokeHandler(Delegate handler, Object target)
   at System.Windows.RoutedEventHandlerInfo.InvokeHandler(Object target, RoutedEventArgs routedEventArgs)
   at System.Windows.EventRoute.InvokeHandlersImpl(Object source, RoutedEventArgs args, Boolean reRaised)
   at System.Windows.UIElement.RaiseEventImpl(DependencyObject sender, RoutedEventArgs args)
   at System.Windows.OleDropTarget.RaiseDragEvent(RoutedEvent dragEvent, Int32 dragDropKeyStates, Int32& effects, DependencyObject target, Point targetPoint)
   at System.Windows.OleDropTarget.MS.Win32.UnsafeNativeMethods.IOleDropTarget.OleDragEnter(Object data, Int32 dragDropKeyStates, Int64 point, Int32& effects)
```

After such error UI is unresponsive until user minimizes and restores application.

**Comment by punker76** [comment]
@Divaite With which version is this happened?

**Comment by Divaite** [comment]
Latest stable 1.0.0

**Comment by punker76** [comment]
@Divaite This is just curious, the `CaptureScreen` method is converting the bounds of `VisualTreeHelper.GetDescendantBounds(target)` with `DpiHelper.LogicalRectToDevice(bounds)` and it seems that it gives us negative size. So what is your DPI OS setting? Or can you explain what do you try to drag (which element)? Thx.

**Comment by Divaite** [comment]
@punker76, OS Windows 7, DPI 96 (100%)
We have a TreeView, in each node representing folder there is a ListBox, each node representing an item is a Grid with Image and TextBlock controls inside.
We use DragDrop with this configuration on ListBox and TreeView:
```
dd:DragDrop.IsDragSource="True"
dd:DragDrop.IsDropTarget="True"
dd:DragDrop.DropHandler="{Binding}"
dd:DragDrop.UseDefaultDragAdorner="True" 
```
It's not easy to catch, but happens when user manipulates with lowest element (Grid with Image and TextBlock).

**Comment by punker76** [comment]
@Divaite I'll make this more robust against negative values, but it's curious to get these negative values from your elements...

**Comment by punker76** [comment]
@Divaite I handle now negative values to prevent this exception (available in v1.1.0). Nevertheless, it would be helpful if you can create a short repro app to demonstrate the issue.
